README.md

```diff
@@ -1 +1,116 @@
 # digital-media-object-processor
+The digital media processor objects can receive digital media object from two different sources:
+- Through the API as a request to register a digital media object
+- Through a Kafka queue to register an annotation
+
+## Preparation
+The digital media object processes the received object as a batch.
```

Comment:
The digital media object processor, processes the received object as a batch?

Reply:
it should be objects (plural), no?
no comma after "processor"